content/en/continuous_integration/static_analysis/default_rules/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Default Rules
+kind: documentation
+type: static_analysis_rules
+description: View rules for languages such as Python for Datadog Static Analysis.
+is_beta: true
+disable_sidebar: true
```

Comment:
hmmm, a sidebar would let the reader a) switch sites, b) quickly hop down to code style or security sections

Reply:
```suggestion
```